packages/shared/lib/core/app/constants/default-app-settings.constant.ts

```diff
@@ -0,0 +1,11 @@
+import { AppTheme } from '../enums'
+import { IAppSettings } from '../interfaces'
+
+export const DEFAULT_APP_SETTINGS: IAppSettings = {
+    deepLinking: true,
+    language: 'en',
+    theme: AppTheme.Light,
+    darkMode: false,
+    notifications: true,
+    sendCrashReports: true,
```

Comment:
Just checked this via `yarn start`. Newly created profiles still have a false value for the "Crash reports" setting (Send crash reports is untoggled)

Reply:
Thats because this is app settings so only works at fresh install, or if you delete at minimum the settings from the application data